src/registrar/templates/base.html

```diff
@@ -65,7 +65,7 @@
       <div class="usa-alert usa-alert--warning usa-alert--no-icon">
       <div class="usa-alert__body">
         <p class="usa-alert__text">
-        <strong>BETA SITE:</strong> We’re building a new way to get a .gov. Take a look around, but don’t rely on this site yet. This site is for testing purposes only. Don’t enter real data into any form on this site. To learn about requesting a .gov domain, visit <a href="https://get.gov" class="usa-link">get.gov</a>
+        <strong>BETA SITE:</strong> We built a new way to request and manage .gov domains—a new .gov registrar. Take a look around, but don’t rely on this site yet. This site is for testing purposes only. Don’t enter real data into any form on this site. Visit <a href="https://get.gov" class="usa-link">get.gov</a> for the latest updates on the new .gov registrar.
```

Comment:
Just want to confirm, this banner will be present on manage.get.gov and beta.get.gov, correct?

Reply:
@dave-kennedy-ecs and @gabydisarli I think this is the figma inteneded for November and things just got mixed up:  ![image](https://github.com/cisagov/manage.get.gov/assets/109625347/692ebc67-cea8-492d-812a-7112ac1c3d48)

gaby can you confirm the image above is the desired result of this ticket?

Reply:
The work done in this PR was for the pre-cutover #1290  (meaning pre-launch) update, which is great. The image you have here is for the post-cutover work #1143

Reply:
I believe we want 1143 in this sprint, and 1290 wasn't set to be in the sprint. I vote we change this text to be that of 1143 as 1290 includes beta.get.gov which is not being changed here.

Reply:
I'm sorry, but I am still confused as to what language I should have in this PR.  This PR is for the banner for manage.get.gov.

Reply:
Yes this PR is for the banner for manage.get.gov. but for launch we want the language in the snippet I added above. It was just to the right of the one you used, so it was easily mixed up. The work you did doesn't meet the AC for 1143.

Reply:
That's fine with me, Alysia! Sorry for all the confusion.